includes/wikia/helpers/CrossOriginResourceSharingHeaderHelper.php

```diff
@@ -88,14 +107,26 @@ private function setResponseHeader( WikiaResponse $response, $headerName, $value
 		}
 	}
 
-	/**
-	 * @return $this
-	 */
-	public function readConfig() {
-		global $wgCORSAllowOrigin;
+	public function allowWhitelistedOrigins( Array $additionalOrigins = [] ) {
+		global $wgWikiaEnvironment, $wgCORSAllowOrigin;
+
+		switch ( $wgWikiaEnvironment ) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement with "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1301)